CMakeModules/NSIS.template.in

```diff
@@ -166,31 +168,31 @@ FunctionEnd
 ##############################################################################
 
 !macro CheckForProcess processName gotoWhenFound gotoWhenNotFound
-   Processes::FindProcess ${processName}
-   StrCmp $R0 "0" ${gotoWhenNotFound} ${gotoWhenFound}
+   Processes::FindProcess "${processName}"
+   StrCmp $R0 "0" "${gotoWhenNotFound}" "${gotoWhenFound}"
 !macroend
 
-!macro ConfirmEndProcess processName
+!macro ConfirmEndProcess processTitle processName
    MessageBox MB_YESNO|MB_ICONEXCLAMATION \
      "Found ${processName} process(s) which need to be stopped.$\nDo you want the installer to stop these for you?" \
-     IDYES process_${processName}_kill IDNO process_${processName}_ended
-   process_${processName}_kill:
+     IDYES "process_${processTitle}_kill" IDNO "process_${processTitle}_ended"
+   process_${processTitle}_kill:
       DetailPrint "Killing ${processName} processes."
-      Processes::KillProcess ${processName}
+      Processes::KillProcess "${processName}"
       Sleep 1500
-      StrCmp $R0 "1" process_${processName}_ended
+      StrCmp $R0 "1" "process_${processTitle}_ended"
       DetailPrint "Process to kill not found!"
-   process_${processName}_ended:
+   process_${processTitle}_ended:
 !macroend
 
-!macro CheckAndConfirmEndProcess processName
-   !insertmacro CheckForProcess ${processName} 0 no_process_${processName}_to_end
-   !insertmacro ConfirmEndProcess ${processName}
-   no_process_${processName}_to_end:
+!macro CheckAndConfirmEndProcess processTitle processName
+   !insertmacro CheckForProcess "${processName}" 0 "no_process_${processTitle}_to_end"
+   !insertmacro ConfirmEndProcess "${processTitle}" "${processName}"
+   no_process_${processTitle}_to_end:
 !macroend
 
 Function EnsureTomahawkShutdown
-   !insertmacro CheckAndConfirmEndProcess "tomahawk.exe"
+   !insertmacro CheckAndConfirmEndProcess "${TARGET_NAME}" "${APPLICATION_NAME}.exe"
```

Comment:
Yup, the first should be for the user visible confirm notification and the second for actually killing the process.